egs2/dirha_wsj/asr1/asr.sh

```diff
@@ -0,0 +1 @@
+../../TEMPLATE/asr1/asr.sh
```

Comment:
Please add the BLSTM performance with wordlm to RESULTS. Meanwhile I will run this to see if you have missed something.

Reply:
I am not sure whether word LM has been implemented, since egs2/Template/asr.sh wrote" log Word LM is not supported yet". If it has been down, I will run the experiments ASAP. @kamo-naoyuki , do you know whether word LM is good now?

Reply:
Sorry, word lm is not supported.
Espnet2 uses v2 decoding module, but it doesn't support wordLM yet.

Reply:
@sas91 , I just added one result to README.md, which is not as good as espnet1. Since it didn't use word lm, the results comparison is not fair. I was wondering whether I need to run the other three mics experiments w/o word lm.